src/scss/_hidden.scss

```diff
@@ -1,5 +1,6 @@
-// taken from Bootstrap
-.jazz-sr-only, .jazz-visually-hidden {
+.jazz-sr-only,
+.jazz-visually-hidden,
+.jazz-visually-hidden:not(:focus):not(:active) {
```

Comment:
I think that the intent of the "not(:focus):not(:active)" part is that the styles will be removed if the element somehow receives focus.  You would have to test it to be sure, but I think that the selector should be something like this:

.jazz-sr-only:not(:focus):not(:active), .jazz-visually-hidden:not(:focus):not(:active)

I think you would want to remove the .jazz-sr-only and .jazz-visually-hidden.  Otherwise, I would expect the styles to still be applied regardless of the focus.  

Also, maybe we should settle on just one of these selectors instead of allowing both.  I would vote for just using "jazz-visually-hidden" and eliminate the use of jazz-sr-only.  You would have to replace references throughout the project.